### ISSUE TITLE
[Feature Request] All replies of a specific message 

### ISSUE THREAD

**Issue by SmartManoj** [telegramdesktop/tdesktop#6141]
### Is your feature request related to a problem?

Hard to find those messags

### Describe the solution you'd like

Hide all message that does not have the reply_to_id of the msg

**Comment by thomas-ashcraft** [comment]
Just came here with the same idea. If totally different people imagined exact the same improvement, it's became more common sense than just a feature request, and thus should be implemented.

In really active and big chats sometimes happens some good questions and acquiring answers for them - is priceless.

For example: someone asked which model of device more suited for a specific circumstances in a chat with about 1000 active members. That someone got answers. But only once, when notifications popups. Then even questioneer can seek his answers only through full text search.

So basically we just need a button to somehow show only those messages, that came as reply for a specified message. "Show replies". Simple and brilliantly.

**Comment by SmartManoj** [comment]
@thomas-ashcraft seems it doesn't react for your reaction

**Comment by SmartManoj** [comment]
We can have threads now for linked groups only.

Describe the solution you'd like
Implement the same thread feature locally
